horace_core/sqw/PixelData/@PixelDataBase/PixelDataBase.m

```diff
@@ -308,14 +308,14 @@
             % indices   -- the indices corresponding to the fields
             %
 
-            if istext(fld_name)
-                idx = PixelDataBase.FIELD_INDEX_MAP_(fld_name);
-            elseif isnumeric(fld_name)
-                idx = fld_name(:)';
-            elseif iscell(fld_name)
+            if iscell(fld_name)
```

Comment:
Unsure why this is being reordered here.

Reply:
was playing with extended `istext` here but decided not to bother.